lib/Conversion/HWToBTOR2/HWToBTOR2.cpp

```diff
@@ -884,9 +884,9 @@ struct ConvertHWToBTOR2Pass
       auto initialConstant = circt::seq::unwrapImmutableValue(init)
                                  .getDefiningOp<hw::ConstantOp>();
       if (!initialConstant)
-        reg->emitError("PowerOn Value must be constant!!");
+        reg->emitError("Initial value must be constant!!");
```

Comment:
Maybe no exclamation marks 😁?
```suggestion
        reg->emitError("initial value must be constant");
```

Reply:
Sounds wise to me!